Foundation-delegation-program/validator-profiles/OriginStake/OriginStake-pledger.md

```diff
@@ -6,12 +6,20 @@ As the Aura foundation delegation receiver, **OriginStake** pledges to meet the
 4. Commission rate is greater than 0% and smaller or equal to 8%
```

Comment:
I mean update here

Reply:
> I mean update here

I have adjusted it more accurately. Thank you for reminding me 🤝